src/components/DeviceStateOverviewItem.js

```diff
@@ -0,0 +1,27 @@
+// @flow
+
+import type { DeviceStatus } from 'brewskey.js-api';
+
+import * as React from 'react';
+import OverviewItem from '../common/OverviewItem2';
+
+const DESCRIPTION_BY_DEVICE_STATE: { [key: DeviceStatus]: string } = {
+  Active: 'Your device is currently active and can pour beers.',
+  Inactive: 'Your device is currently inactive and cannot pour beers.',
+  Unlocked:
```

Comment:
This status isn't true anymore. I think I've updated these in the other branch. There is also `Cleaning`

Reply:
don't you confuse with `free` ? 
```
<EnumType Name="DeviceStatus">
<Member Name="Active" Value="1"/>
<Member Name="Inactive" Value="2"/>
<Member Name="Cleaning" Value="3"/>
<Member Name="Unlocked" Value="4"/>
</EnumType>```
in the current schema

..I know about `cleaning`, but  there is no description for this state in the other branch.

Reply:
Ahh.. it's in `new-device.html` I need to update that page too.